nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenHTTP.java

```diff
@@ -193,7 +193,7 @@
             .description("Client Authentication policy for TLS connections. Required when SSL Context Service configured.")
             .required(false)
             .allowableValues(ClientAuth.values())
-            .defaultValue(ClientAuth.REQUIRED.name())
```

Comment:
I don't think we want to make this change. The default value should always lean toward being more secure, not less secure. And if I set an SSL Context on the processor, I would assume mutual auth is going to happen unless i explicitly change it. This is especially true, since that's always been the behavior in the past.

Reply:
Previous behavior inferred requiring a client certificate when the `SSLContextService` was configured with trust store properties, falling back to wanting a client certificate in the absence of trust store properties.  Making `REQUIRED` the default value going forward sounds like the best approach.  Perhaps putting a comment in future release notes indicating this change in behavior would be worthwhile for users of `ListenHTTP` who do not require clients to provide certificates.